element_array_ephys/ephys_report.py

```diff
@@ -139,14 +153,14 @@ def make(self, key):
 def _make_save_dir(root_dir: pathlib.Path = None) -> pathlib.Path:
     if root_dir is None:
         root_dir = pathlib.Path().absolute()
-    save_dir = root_dir / "ephys_figures"
+    save_dir = root_dir / "temp"
```

Comment:
The gitignore will skip all `temp*` files, so this will add specificity for users
```suggestion
    save_dir = root_dir / "temp_ephys_figures"
```